components/speedreader/BUILD.gn

```diff
@@ -37,8 +35,11 @@ source_set("speedreader") {
   deps = [
     "//brave/components/brave_component_updater/browser",
     "//brave/components/resources",
+    "//brave/components/resources:static_resources_grit",
+    "//components/prefs:prefs",
     "//services/network/public/cpp",
     "//services/network/public/mojom",
+    "//third_party/blink/public/common:headers",
```

Comment:
I'm confused, you removed the header, but added a dep. In any case this dep is not allowed and adding it to visibility list is the wrong thing to do because this runs in the browser process and blink deps are not allowed there.

Reply:
I removed "url_loader_throttle.h" include in "speedreader_url_loader.h" file because it looks like it's unnecessary. But, "url_loader_throttle.h" is still being included by speedloader_throttle.h file. That's why I added this dependency. And also, I found //chrome/browser has "//third_party/blink/public/common" deps as well as "//third_party/blink/public/common" is also opened to "//chrome/*". So, I thought "blink public" could be used by this as well as allow to add this to the visibility list. If I misunderstood about it, thanks for pointing it out.

Anyway, as I mentioned, speedreader_throttle.h file is still including "third_party/blink/public/common/loader/url_loader_throttle.h" file. So, IMHO, I still need to solve this dependency violation. Any idea to fix it?

[1] https://source.chromium.org/chromium/chromium/src/+/master:chrome/browser/BUILD.gn;l=2126?q=%22%2F%2Fthird_party%2Fblink%2Fpublic%2Fcommon%22,&ss=chromium&originalUrl=https:%2F%2Fcs.chromium.org%2F

Reply:
you are right, `third_party/blink/public/common:headers` does seem to be ok for chrome/browser, but we can avoid the patch by adding a dep for `//third_party/blink/public/common` which does not have a visibility list and has a public dep for `"//third_party/blink/public/common:headers",`
```
# Public common API headers, mojom and libraries that can be linked and
# referenced both by browser-side and renderer-side components.
```

Reply:
ok, I add //third_party/blink/public/common to the deps instead of "//third_party/blink/public/common:headers". Thanks!